hvat/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "hvat"
-version = "0.6.2"
+version = "0.7.0"
```

Comment:
@ramijebara I debated jumping to 1.0.0, but I think we're in a grey area--there should technically be no breaking changes for the end user.

Let me know if you'd prefer something different here.

Reply:
0.7.0 is good in this case.